draft-ietf-quic-transport.md

```diff
@@ -6977,6 +6980,23 @@ particular ingress filtering {{?BCP38=RFC2827}}, are especially effective
 against attacks that use spoofing and originate from an external network.
 
 
+### Request Forgery with Version Negotiation {#vn-spoofing}
+
+Clients that are able to present a spoofed source address on a packet can cause
+a server to send a Version Negotiation packet {{packet-version}} to that
+address.
+
+The absence of size restrictions on the connection ID fields for packets of an
+unknown version increases the amount of data that the client controls from the
+resulting datagram.  The first byte of this packet is not under client control
+and the next four bytes are zero, but the client is able to control up to 512
+bytes starting from the fifth byte.
+
+No specific countermeasures are provided for this attack, though generic
+protections {{forgery-generic}} could apply.  In this case, ingress filtering
+{{?BCP38}} is also effective.
+
```

Comment:
I have been thinking about potential defenses, and I could not find something obvious there. Rate limiting does not help much, since this is not a volumetric attack. I think this text is about the best we can do.